styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -120,12 +121,14 @@
   public static final String STYX_MODE_DEVELOPMENT = "development";
 
   public static final int SCHEDULER_TICK_INTERVAL_SECONDS = 2;
+  public static final int SUBMISSION_RATE_LIMITER_UPDATE_TICK_INTERVAL_SECONDS = 5;
```

Comment:
It could look like the current approach generalized. Something like `startRuntimeConfigUpdate(Storage configStorage, RateLimiter rateLimiter, Map<String, Resource> resources, SomeOtherUpdatableThatWeCareAbout foo, ..., ScheduledExecutorService exec)` would be fine IMHO.

I just want to ensure that we leave a nice message to the next editor of this code, that conveys that we want to go in consistent direction, not in ad-hoc directions. I don't require refactor in this PR, just rename the unnecessarily specific names to point in that more general direction (makes names more digestible too).